README.md

```diff
@@ -331,6 +331,59 @@ This addon uses CSS variables to customise the styling. You can override these v
 }
 ```
 
+#### Override node classes
+
+You can also customize the look of your editor by either adding styles to the class of each node, or even changing that class to your custom one.
+The classes asigned to each node are the following:
+| Node | Css class |
+|---|---|
+| Heading | say-heading |
+| Paragraph | say-paragraph |
+| RepairedBlock | say-repaired-block |
+| ListItem | say-li |
```

Comment:
As all others classnames are written in full, maybe we could also use `say-list-item` here? Really pedantic I know.